src/extension.ts

```diff
@@ -391,54 +391,30 @@ export class Extension implements RunHooks {
     }
   }
 
-  private async _runTests(include: readonly vscodeTypes.TestItem[] | undefined, mode: 'run' | 'debug' | 'watch') {
+  private async _runTests(request: vscodeTypes.TestRunRequest, mode: 'run' | 'debug' | 'watch') {
     this._completedSteps.clear();
     this._executionLinesChanged();
 
-    // Create a test run that potentially includes all the test items.
```

Comment:
This can be removed since https://github.com/microsoft/vscode/issues/213182 has been landed upstream.